raiden-ts/tests/unit/epics/path.spec.ts

```diff
@@ -11,24 +11,28 @@ import {
   channelOpen,
   channelDeposit,
   channelClose,
+  channelMonitor,
 } from 'raiden-ts/channels/actions';
-import { raidenConfigUpdate } from 'raiden-ts/actions';
+import { raidenConfigUpdate, RaidenAction } from 'raiden-ts/actions';
 import { matrixPresence } from 'raiden-ts/transport/actions';
 import { raidenReducer } from 'raiden-ts/reducer';
 import {
   pathFindServiceEpic,
   pfsCapacityUpdateEpic,
   pfsServiceRegistryMonitorEpic,
+  pfsFeeUpdateEpic,
 } from 'raiden-ts/path/epics';
 import { pathFind, pfsListUpdated, iouPersist, iouClear } from 'raiden-ts/path/actions';
 import { messageGlobalSend } from 'raiden-ts/messages/actions';
 import { MessageType } from 'raiden-ts/messages/types';
 import { losslessStringify } from 'raiden-ts/utils/data';
+import { pluckDistinct } from 'raiden-ts/utils/rx';
+import { ErrorCodes } from 'raiden-ts/utils/error';
```

Comment:
Didn't we have eslint doing import sorting automatically? this change feels like something that would be handled by eslint

Reply:
Not sure if that includes tests, it didn't get sorted automatically by my prettier plugin, so I suppose it's not into effect there. No big deal anyway for tests.

Reply:
It should be a big deal, we should treat our tests the same as our production code. :D 
not for this issue but I will create a note in the iteration to have a look.